package.json

```diff
@@ -4,13 +4,18 @@
   "description": "Standalone WWW client for Scratch",
   "scripts": {
     "start": "node ./dev-server/index.js",
-    "test": "npm run test:lint && npm run build && npm run test:tap",
+    "test": "npm run test:lint && npm run build && npm run test:unit:tap",
     "test:lint": "eslint . --ext .js,.jsx,.json",
-    "test:smoke": "tap ./test/integration/smoke-testing/*.js --timeout=3600 --no-coverage -R classic",
-    "test:smoke:verbose": "tap ./test/integration/smoke-testing/*.js --timeout=3600 --no-coverage -R spec",
-    "test:smoke:sauce": "SMOKE_REMOTE=true tap ./test/integration/smoke-testing/*.js --timeout=60000 --no-coverage -R classic",
-    "test:tap": "tap ./test/{unit,localization}/*.js --no-coverage -R classic",
-    "test:coverage": "tap ./test/{unit,localization}/*.js --coverage --coverage-report=lcov",
+    "test:integration": "npm run test:integration:jest && npm run test:smoke",
+    "test:integration:jest": "jest ./test/integration/*.test.js",
```

Comment:
I think we should have more general paths for most of these, but that could be a follow-on change by me or you

Reply:
I think that should come in a later pr that includes tests organized into sub folders.

Reply:
Great. Maybe make an issue to track that?

Reply:
I filed an issue here: https://github.com/LLK/scratch-www/issues/3157